qml/pages/ChatPage.qml

```diff
@@ -497,7 +497,6 @@ Page {
                             onTriggered: {
                                 webPagePreviewLoader.active = ( typeof display.content.web_page !== "undefined" );
                                 imagePreviewLoader.active = ( display.content['@type'] === "messagePhoto" );
-                                stickerPreviewLoader.active = ( display.content['@type'] === "messageSticker" );
```

Comment:
Sorry, a bit of an inconsistency is introduced here, but I really want stickers to be displayed right away rather than after 500ms timeout.